plugins/arDominionB5Plugin/modules/search/templates/_advancedSearch.php

```diff
@@ -150,116 +167,99 @@
-                    ->label(__('Level of description'))
-                    ->renderRow(); ?>
+              <div class="col-md-4">
+                <?php echo render_field($form->levels->label(__('Level of description'))); ?>
               </div>
 
-              <div class="filter-center">
-                <?php echo $form->onlyMedia
-                    ->label(__('%1% available', ['%1%' => sfConfig::get('app_ui_label_digitalobject')]))
-                    ->renderRow(); ?>
+              <div class="col-md-4">
+                <?php echo render_field($form->onlyMedia->label(__('%1% available', ['%1%' => sfConfig::get('app_ui_label_digitalobject')]))); ?>
               </div>
 
-              <div class="filter-right">
-                <?php echo $form->findingAidStatus
-                    ->label(__('Finding aid'))
-                    ->renderRow(); ?>
+              <div class="col-md-4">
+                <?php echo render_field($form->findingAidStatus->label(__('Finding aid'))); ?>
               </div>
 
             </div>
 
-            <?php $showCopyright = sfConfig::get('app_toggleCopyrightFilter'); ?>
-            <?php $showMaterial = sfConfig::get('app_toggleMaterialFilter'); ?>
+            <div class="row">
 
-            <?php if ($showCopyright || $showMaterial) { ?>
-              <div class="filter-row">
+              <?php if (sfConfig::get('app_toggleCopyrightFilter')) { ?>
+                <div class="col-md-6">
+                  <?php echo render_field($form->copyrightStatus->label(__('Copyright status'))); ?>
+                </div>
+              <?php } ?>
 
-                <?php if ($showCopyright) { ?>
-                  <div class="filter<?php echo $showMaterial ? '-left' : ''; ?>">
-                    <?php echo $form->copyrightStatus
-                        ->label(__('Copyright status'))
-                        ->renderRow(); ?>
-                  </div>
-                <?php } ?>
+              <?php if (sfConfig::get('app_toggleMaterialFilter')) { ?>
+                <div class="col-md-6">
+                  <?php echo render_field($form->materialType->label(__('General material designation'))); ?>
+                </div>
+              <?php } ?>
 
-                <?php if ($showMaterial) { ?>
-                  <div class="filter<?php echo $showCopyright ? '-right' : ''; ?>">
-                    <?php echo $form->materialType
-                        ->label(__('General material designation'))
-                        ->renderRow(); ?>
+              <div class="col-12">
+                <div class="d-grid d-sm-block mt-2">
+                  <div class="form-check d-inline-block me-2">
+                    <input class="form-check-input" type="radio" name="topLod" id="adv-search-top-lod-1" value="1" <?php echo $topLod ? 'checked' : ''; ?>>
+                    <label class="form-check-label" for="adv-search-top-lod-1"><?php echo __('Top-level descriptions'); ?></label>
```

Comment:
Use fieldset and legend ...

https://www.w3.org/WAI/tutorials/forms/grouping/

Or radiogroup role ...

https://www.w3.org/TR/2017/WD-wai-aria-practices-1.1-20170628/examples/radio/radio-1/radio-1.html

Reply:
Thanks!